config/application.rb

```diff
@@ -54,4 +54,4 @@ class Application < Rails::Application
     # Public Safety alert configuration.
     config.x.public_safety.feed = 'https://www.getrave.com/rss/chapman/channel1'
   end
-end
+end
```

Comment:
@cbryant24 It's common best practice to have a newline at the end of files. Is there a reason it was removed here?

Reply:
No reason I was reviewing the github instructions on disabling the Public Safety Alert Configuration, the extra line I must've deleted. I'll add that extra line back in.